DL_AdditionalMessage.cpp

```diff
@@ -0,0 +1,143 @@
+#include <iostream>
+#include <string>
+
+
+enum class MessageIntentUse {
+     DueToWeather,
+     DueToAircraft
+    
+};
+
+enum class MessageElement {
```

Comment:
this class is used at multiple points it's better to make it in parent classes

Reply:
I create DownLink.h file